tasks/html.js

```diff
@@ -18,13 +18,38 @@ module.exports = function(grunt) {
     var done = this.async(),
       files = grunt.file.expand(this.filesSrc),
       options = this.options({
+        wrapPath: '',
         files: files,
         force: false,
         absoluteFilePathsForReporter: false
       }),
       force = options.force,
       reporterOutput = options.reporterOutput,
-      reporter;
+      reporter,
+      lineOffset = 0,
+      partialTmpDir = '.tmp-grunt-html';
+
+    function wrapPartials() {
```

Comment:
This function has multiple issues. I don't have time right now to dig into the details, maybe you can find someone else to do a review for this.